scripts/docs/en/userver/logging.md

```diff
@@ -256,7 +256,55 @@ It means the logger is successfully initialized and is ready to process the logs
 If somethings goes wrong (e.g. OTLP collector agent is not available), you'll see errors in stderr.
 The service buffers not-yet-sent logs and traces in memory, but drops them on overflow.
 
-----------
+## Separate Sinks for Logs and Tracing
+
+In certain environments, such as Kubernetes, applications typically write logs to stdout/stderr, while traces are sent efficiently through the 'push' model (via OTLP transport). Kubernetes stores the container's stdout/stderr in files on nodes, making logs available for log collectors using the 'pull' model. This approach ensures that logs remain accessible even if the application fails, capturing the most critical information.
+
+To configure separate sinks for logs and traces, use the optional 'sinks' block in the 'otlp-logger' configuration (see the full schema in otlp::LoggerComponent::GetStaticConfigSchema):
```

Comment:
The options should be described in the doc comments on the `LoggerComponent` class. Here we should refer to the component and its docs, not to the sources